app/views/commontator/comments/_show.html.erb

```diff
@@ -126,8 +138,8 @@
         <span id="commontator-comment-<%= comment.id %>-votes" class="votes">
           <%= render partial: 'commontator/comments/votes', locals: { comment: comment, user: user } %>
         </span>
-        <% if !comment.parent_id? %>
```

Comment:
Is there a reason for removing this?

Reply:
Good catch Josh - yeah - I think this was removed unintentionally... I can't remember why it was removed.. so I added back in.